src/tally.h

```diff
@@ -49,6 +51,8 @@ namespace pyne
 
     ~Tally (); /// default destructor
 
+    // is the proposed name a valid tally name
+    bool is_valid(std::string s); 
```

Comment:
Perhaps `is_supported` is a better name...  a tally could be theoretically valid and not supported, and a user could use a supported tally, but define it in an invalid way...